package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "markdown-it-mathjax3": "^4.3.2",
     "markdown-it-vuepress-code-snippet-enhanced": "github:IMB11/md-it-enhanced-snippets#dfb9fa2",
+    "medium-zoom": "^1.1.0",
```

Comment:
Question: Whats this used for? Maybe split up the PR a little to make it easier to see what goes where.

Reply:
See [#114](https://github.com/FabricMC/fabric-docs/issues/114)

Click on an image within the content of the page, it will zoom allowing you to view it at a larger width/height. Similar to how medium.com has their image viewer.